chapter1.md

```diff
@@ -67,37 +67,31 @@ Model fitting, log-likelihood, and AICc calculations were performed using the ma
 
 ![AICc weights by model for all datasets combined.](./sad-data/chapter1/AICc_weights.png)
 
-While the AICc weights show separation among models, these values include a correction for the number of parameters and are also influenced by the similarity between models. Therefore, we also compared the negative log-likelihoods of the different models to determine whether or not their absolute fits differed. Frequency distributions of log-likelihoods show almost complete overlap among models (Figure 4) and one-to-one plots of the likelihoods of each model against the likelihood of the log-series show that the likelihoods of the different models correspond almost perfectly for individual distributions (Figure 5). This indicates that all models fit the data equivalently and that differences in AICc weights resulted primarily from differences in the number of parameters and differences in how similar different models in the set of models were (i.e., if three identically fitting models are included in the analysis none of them can have a AICc weight > 0.34).
-
 ![Log-likelihoods by model for all datasets combined.](./sad-data/chapter1/likelihoods.png)
 
 ![Log-likelihoods by model for all datasets combined.](./sad-data/chapter1/likelihoods_one_to_one.png)
 
 # Discussion
 
-Our extensive comparison of different models for the species abundance distribution (SAD) using rigorous statistical methods demonstrates that most existing models provide equivalently good absolute fits to empirical data. As a result, the models with the fewest parameters perform better in AIC-based model selection since these approaches penalize model complexity. Since the log-series provides equivalent likelihoods to the other distributions, has a single fitted parameter, is easy to fit to empirical data, and is the best overall model using standard model selection, it provides a good naive model for fitting SADs.
-
-The similar absolute fits of these five commonly used distributions emphasizes the challenges of inferring the processes operating in ecological systems from the form of the abundance distribution. It is already well established that models based on different processes can yield equivalent models of the SAD, i.e., they predict distributions of exactly the same form [@cohen1968, @pielou1975, @boswell1971, @mcgill2007]. It is also possible for the same biological explanations to result in  different forms of the species abundance distribution depending on community conditions [@hughes1986]. Our results support the idea that even when models do differ in their precise mathematical predictions that they are often not distinguishable enough to identify potential mechanisms with any degree of certainty [@volkov2005]. In other words, it is difficult to distinguish among the different distributions used to characterize the SAD, let alone the processes that generate the form of a particular distribution.
```

Comment:
I'm wondering if we want to discard or keep these paragraphs on SAD vs mechanism. Dave, what are your thoughts?